src/Libraries/CoreNodeModels/Enum.cs

```diff
@@ -12,6 +12,11 @@ namespace CoreNodeModels
 {
     public abstract class EnumAsInt<T> : EnumBase<T>
     {
+        protected EnumAsInt() { }
```

Comment:
please add a comment where you needed to add these.

Reply:
see [this comment](https://github.com/DynamoDS/DynamoRevit/pull/1783#discussion_r138064533) in the D4R portion of the PR